src/main/java/com/pokegoapi/auth/PtcLogin.java

```diff
@@ -166,7 +166,7 @@ public AuthInfo login(String username, String password) throws LoginFailedExcept
 					.addQueryParameter("client_id", CLIENT_ID)
 					.addQueryParameter("redirect_uri", REDIRECT_URI)
 					.addQueryParameter("client_secret", CLIENT_SECRET)
-					.addQueryParameter("grant_type", "refresh_token")
+					.addQueryParameter("grant_type", "refreshToken")
```

Comment:
the official client is using refresh_token so will should keep it .addQueryParameter("grant_type", "refresh_token")